lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -154,6 +142,63 @@ export abstract class BaseManagedIdentitySource {
             }
         }
 
+        let serverTokenResponse: ServerAuthorizationTokenResponse =
```

Comment:
It seems to me that the RetryTracker object can be created directly in this method only. There's no need to keep it as class member in all these classes.